histogram/src/histogram.rs

```diff
@@ -461,6 +461,19 @@ impl Clone for Histogram {
     }
 }
 
+impl TryFrom<&CompactHistogram> for Histogram {
+    type Error = error::Error;
+
+    fn try_from(h: &CompactHistogram) -> Result<Self, Self::Error> {
+        let histogram = Histogram::new(h.m, h.r, h.n)?;
+        for (i, bucket_id) in h.index.iter().enumerate() {
+            let v = h.count[i];
+            histogram.buckets[*bucket_id].store(v, Ordering::Relaxed);
```

Comment:
Should we use `.get(idx)` here anyway? I know that the number of buckets *should* match, but is that enforced? Is a panic here the right way to handle that?

Reply:
Good catch, I'll make a fix.